src/build/common/tasks/publish.yaml

```diff
@@ -1,12 +1,16 @@
+# Task to publish a path as an artifact
 parameters:
-  path: ''
-  artifactName: '$(Build.DefinitionName)'
-  displayName: 'Artifact Publish'
+  - name: path
+    default: ''
```

Comment:
this should be required IMO, or default to $(build.ArtifactStagingDirectory) - what do you think?

Reply:
I think this is fine as empty, the `PathToPublish` on line 14 is already set to include `build.ArtifactStagingDirectory`. So if you want to publish the root staging directory, you configure nothing, if you need a sub-folder, you only specify the sub-folder?